pygeos/test/test_io.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+import pygeos
+import pytest
+
+from .common import all_types
+
+
+POINT11_WKB = (
+    b"\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\xf0?"
+)
+
+
+def test_from_wkt():
+    expected = pygeos.points(1, 1)
+    actual = pygeos.from_wkt("POINT (1 1)")
+    assert pygeos.equals(actual, expected)
+    # also accept bytes
+    actual = pygeos.from_wkt(b"POINT (1 1)")
+    assert pygeos.equals(actual, expected)
+
+    # None propagates
+    assert pygeos.from_wkt(None) is None
+
+    with pytest.raises(TypeError, match="Expected bytes, got int"):
+        pygeos.from_wkt(1)
+
+    with pytest.raises(pygeos.GEOSException):
+        pygeos.from_wkt("")
+
+    with pytest.raises(pygeos.GEOSException):
+        pygeos.from_wkt("NOT A WKT STRING")
+
+
+@pytest.mark.parametrize("geom", all_types)
+def test_from_wkt_all_types(geom):
+    wkt = pygeos.to_wkt(geom)
+    actual = pygeos.from_wkt(wkt)
+    assert pygeos.equals(actual, geom)
+
+
+@pytest.mark.parametrize(
+    "wkt", ("POINT EMPTY", "LINESTRING EMPTY", "GEOMETRYCOLLECTION EMPTY")
+)
+def test_from_wkt_empty(wkt):
```

Comment:
Can you also make this test for the WKB?  I vaguely remember segfaults when trying to serialize and deserialize empty geometries from an to WKB (especially the POINT EMPTY)